test/lit/passes/type-ssa-shared.wast

```diff
@@ -86,7 +86,7 @@
   ;; CHECK:      (rec
   ;; CHECK-NEXT:  (type $A_1 (sub $A (shared (array (mut i32)))))
 
-  ;; CHECK:       (type $4 (struct (field (mut i32)) (field (mut i32)) (field (mut f64)) (field (mut f64)) (field (mut i32)) (field (mut f64)) (field (mut f64)) (field (mut i32)) (field (mut i32)) (field (mut i32)) (field (mut i32))))
+  ;; CHECK:       (type $4 (struct))
```

Comment:
Why did this change?

Reply:
TypeSSA no longer encodes a hash as a brand type, but rather uses the brand iterator, which produces much smaller types.